src/main/reactjs/src/components/admin/user/MemberChatSearch.js

```diff
@@ -1,9 +1,64 @@
-import React from 'react';
-import { useNavigate } from 'react-router-dom';
-import { Link } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { Link, useLocation, useNavigate } from 'react-router-dom';
+import axios from "axios";
+import { Pagination, InputAdornment, OutlinedInput } from '@mui/material';
+import SearchIcon from '@mui/icons-material/Search';
+
 
 const MemberChatSearch = () => {
     const nav = useNavigate();
+    const [chat, setChat] = useState([]);
+    const [loading, setLoading] = useState(false);
+    const [searchQuery, setSearchQuery] = useState('');
+    const location = useLocation();
+    const params = new URLSearchParams(location.search);
+    const usercode = params.get("usercode");
+    const [member, setMember] = useState([]);
+    const [currentPage, setCurrentPage] = useState(1);
+    const [itemsPerPage] = useState(4);
+
+    useEffect(() => {
+        if (usercode) {
+            getMember(usercode);
+            fetchChat(usercode);
+        }
+    }, [usercode]);
+
+    const getMember = () => {
+        const url = "/member/data?usercode=" + usercode;
+        axios.post(url, {})
+            .then(res => {
+                setMember(res.data);
+            })
+    }
+
+    const fetchChat = (usercode) => {
+        setLoading(true);
+        axios.get(`/chat/list?usercode=${usercode}`)
+            .then(res => {
+                setChat(res.data);
+            })
+            .catch(error => {
+                console.error("포인트 사용내역을 불러오는 중 오류 발생:", error);
```

Comment:
```suggestion
                console.error("채팅 내역을 불러오는 중 오류 발생:", error);
```
사소하지만 에러 확인 문구를 변경하실 필요가 있을 것 같습니다!